internal/api/openapi/rhtas-console.yaml

```diff
@@ -284,6 +284,115 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /api/v1/trust/targets:
+    get:
+      summary: Get TUF Targets List
+      description: |
+        Fetches and returns the list of TUF target files from the specified repository.
+      tags:
+        - Trust
+      parameters:
+        - name: tufRepositoryUrl
```

Comment:
I'm wondering whether we should make this parameter optional. A configured instance of the dashboard backend should know what TUF repo it's tied to, right? (Assuming this API is only supposed to be used by the frontend which will automatically add the right URL, I guess this is fine; we can go from required to optional any time in the future)